test/DebugInfo/modulecache.swift

```diff
@@ -15,7 +15,8 @@ import ClangModule
 
 // RUN: %empty-directory(%t)
 // RUN: %target-swift-frontend %s -c -g -o %t.o -module-cache-path %t -I %S/Inputs
-// RUN: file %t/*/ClangModule-*.pcm | egrep -q '(Mach-O|ELF)'
+// RUN: llvm-readobj -h %t/*/ClangModule-*.pcm | %FileCheck %s
+// CHECK: Format: {{(Mach-O|ELF|COFF)}}
```

Comment:
There's another invocation of grep on line 12, too.

Reply:
I'll fix that up in a follow up PR since that change would be unrelated to the purpose of this PR.